versa_system/versa_system/custom_script/quotation.py

```diff
@@ -17,9 +17,9 @@ def set_missing_values(source, target):
 
     def filter_approved_items(source, target, source_parent):
         # Only map rows where the 'approve' checkbox is checked
-        if source.approve:
```

Comment:
why was this removed?

Reply:
The 'ItemDetails' object has no attribute 'approve' so I removed that condition, there is no need for that condition